scripts/check_codersdk_imports.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# This file checks all codersdk imports to be sure it doesn't import any packages
+# that are being replaced in go.mod.
+
+set -eu
+# shellcheck source=scripts/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
+cdroot
+
+set +e
+deps=$(./scripts/list_dependencies.sh github.com/coder/coder/v2/codersdk)
+replaces=$(grep "^replace" go.mod | awk '{print $2}')
+conflicts=$(echo "$replaces" | grep -xF -f <(echo "$deps"))
+
+if [ -n "${conflicts}" ]; then
+	error "$(printf 'codersdk cannot import the following packages being replaced in go.mod:\n%s' "${conflicts}")"
```

Comment:
I don't think you need to use printf here just using `error` should be sufficient
```suggestion
	error "codersdk cannot import the following packages being replaced in go.mod: ${conflicts}"
```

Reply:
There's no trailing newline on `conflicts`, and I wanted it to not print the first item on the same line as the message